bin/gollum-migrate-tags

```diff
@@ -189,16 +218,21 @@ class ::Gollum::Filter::TagMigrator < Gollum::Filter::Tags
   private
   
   def tag_for_pick(pick, orig_tag, extra, anchor, linking_page_path)
-    pick = if defined?(PREFER_RELATIVE)
+    pick = if const_true(:PREFER_RELATIVE)
       overlapping_path = Pathname.new(linking_page_path).dirname.to_s
-      relative_path = pick.to_s.match(/^\/#{overlapping_path}\/(.+)/)
+      if overlapping_path == '.'
+        overlapping_path = '' 
+      else
+        overlapping_path = ::File.join('/', overlapping_path)
+      end
+      relative_path = pick.to_s.match(/^#{overlapping_path}\/(.+)/)
```

Comment:
Did the Pathname solution I proposed not work properly?

Reply:
I couldn't get it to work sadly. The problem is `pick` has to start with a `/`, and so is absolute. But `linking_page_path` is relative. This triggers an `ArgumentError` in `Pathname.relative_path_from`. Making `linking_page_path` absolute as well has the consequence that the result of `relative_path_from` will also be absolute. Making `pick` relative as well means you sometimes end up with paths like `../Foo/Bar.md` (that is, *all* generated links become relative to root).